src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: $card-border-width solid #f3f3f3;
+  padding: $card-padding;
+
+  &__image {
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+
+    @include blocks-width;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+    margin: 4px 0 0;
+
+    @include blocks-width;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    @include blocks-width;
+
+    &_label {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $second-text-color;
+    }
+
+    &_value {
```

Comment:
Similar to the previous comment, verify that `&_value` is correctly nested under `&__price`. If it should be a separate element, you might want to adjust its nesting.